docs/libraries/http/reference/data-types.md

```diff
@@ -158,6 +242,15 @@ Implicit flow
 model TypeSpec.Http.ImplicitFlow
 ```
 
+#### Properties
+
+| Name             | Type       | Description                       |
+| ---------------- | ---------- | --------------------------------- |
+| type             | `implicit` | implicit flow                     |
+| authorizationUrl | `string`   | the authorization URL             |
+| refreshUrl?      | `string`   | the refresh URL                   |
+| scopes           | `Array`    | list of scopes for the credential |
```

Comment:
Should array say what it is an array of? Here I would assume it's strings.

Reply:
yeah it probably should but I think that's all part of improving the `getSignature()` and `getTypeName` helper